src/compaction_filter.h

```diff
@@ -31,15 +31,15 @@ class TitanCompactionFilter final : public CompactionFilter {
 
   const char *Name() const override { return filter_name_.c_str(); }
 
-  Decision FilterV2(int level, const Slice &key, ValueType value_type,
-                    const Slice &value, std::string *new_value,
-                    std::string *skip_until) const override {
+  Decision FilterV2(int level, const Slice& key, ValueType value_type,
```

Comment:
use V3?

Reply:
We can only call `FilterV2` in `FilterV3` because some users maybe have re-implemented `FilterV2`.